services/taskcluster-github/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout:       default
+class:        html
+docson:       true
+marked:       true
+ejs:          true
+superagent:   true
+docref:       true
+---
+
+<h1>TaskCluster GitHub</h1>
+<p>
+Easily trigger TaskCluster jobs based on GitHub pushes and pull requests. Tasks are defind in a YAML coniguration file which lives at the root of a repository. There is no explicit sign-up step, TaskCluster and Mozilla projects will simply begin accepting jobs as soon as a <code>.taskclusterrc</code> exists.
```

Comment:
You may want to elaborate below on what "projects" means in this case, and that it can be enabled for other orgs.